### DISCUSSION TITLE
I keep getting this error: "   WARNING  No search models loaded. Configure a search model before"

### DISCUSSION THREAD

**Discussion by LizzardKing94** [khoj-ai/khoj#277] [General]
Hi, 
thank you for such a wonderful idea and taking the time to develop this plugin. Sadly i cant get it to work :( after following all the installation steps i keep getting this error in terminal:
   WARNING  No search models loaded. Configure a search model before        api.py:180
                    initiating search 

can you please tell me what i should do to fix it?

**Comment by LizzardKing94** [comment]
edit:
not im getting this error 
  ERROR    'utf-8' codec can't decode byte 0xff in position 0: invalid     api.py:361
                    start byte

**Comment by debanjum** [comment]
Hi @LizzardKing94, glad you liked the idea, it's unfortunate that your khoj setup did not go smoothly. Could you share more details on
1. what steps have you taken to install khoj
2. screenshots of the error you are seeing
3. The operating system and the surface you want to use Khoj from (e.g Obsidian, Emacs, Web)
